ETC.tsv

```diff
@@ -252,14 +252,14 @@ ETC_20150317_000251	Blue Sequoia
 ETC_20150317_000252	Corrupted
 ETC_20150317_000253	Blue Corrupted
 ETC_20150317_000254	Bagworm
-xETC_20150317_000255	Bagworms are of rare occurence, yet the monsters have little concern about possible extinction given their current numbers.
+xETC_20150317_000255	Bagworms are rare given their current numbers, yet the monsters show little concern about possible extinction.
```

Comment:
Try:
Bagworms are a rare occurence given their current numbers, yet the monsters show little concern about possible extinction.